exim/config/aliases.sls

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+
+{#- Get the `tplroot` from `tpldir` #}
+{%- set tplroot = tpldir.split('/')[0] %}
+{%- set sls_package_install = tplroot ~ '.package.install' %}
+{%- from tplroot ~ "/map.jinja" import exim with context %}
+
+{%- set virtual_aliases = salt['pillar.get']('exim:virtual_aliases', {}) %}
```

Comment:
I wonder, do we really need to fetch data directly from Pillar every time? This would cause additional load on master and doesn't look necessary.
How about to get from `exim` dictionary, that has already merged config, Pillar, and defaults? I.e.
```jinja
 {%- set virtual_aliases = exim.virtual_aliases|default({}) %}
```

I think this could be applied to other similar places.